### PR TITLE
Clear dotenvs after every test execution, solve #79

### PR DESCRIPTION
**Pull request by CristhianMotoche**
@juanpaucar Could you take a look, please? To solve #79

### CODE REVIEW

File: spec/Configuration/DotenvSpec.hs [hunk 1]
```diff
@@ -120,13 +120,22 @@ spec = do
       liftM (!! 4) (parseFile "spec/fixtures/.dotenv") `shouldReturn`
         ("ME", me)
 
-  describe "onMissingFile" $ after_ (unsetEnv "DOTENV") $ do
+  describe "onMissingFile" $ after_ (mapM_ unsetEnv dotenvFixtures) $ do
```
Comment (juanpaucar) [inline spec/Configuration/DotenvSpec.hs]: why not extract this to a function `mapM_ unsetEnv dotenvFixtures` too?

**Comment by CristhianMotoche** [comment]
@juanpaucar I change the perspective of the PR. Now, I clean the entire environment until every test execution. For the `loadFile` tests, I clean the environment and then set up the necessary envs. To get the username, I run the command `whoami` and extract the text result. Then, I compare the result text with the expected result, which should be the same. WDYT?

**Review by juanpaucar — APPROVED** [review]
LGTM thanks

**Comment by CristhianMotoche** [comment]
Thanks to you @juanpaucar

**Comment by sternenseemann** [comment]
This can break the test suite completely since it unsets `PATH` which may lead to `shell "whoami"` which is used in a couple of places may fail. Probably the simplest alternative is to use [getLoginName](https://hackage-content.haskell.org/package/unix-2.8.7.0/docs/System-Posix-User.html#v:getLoginName) since it seems to assume Unix anyways.

**Pull request merged**